adapters/rubicon/rubicon.go

```diff
@@ -537,13 +399,16 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *ada
 			videoCopy.Ext, err = json.Marshal(&videoExt)
 			imp.Video = &videoCopy
 			imp.Banner = nil
-		} else {
-			primarySizeID, altSizeIDs, err := parseRubiconSizes(imp.Banner.Format)
-			if err != nil {
-				errs = append(errs, err)
+			imp.Native = nil
+		} else if imp.Banner != nil {
+			var banner = *imp.Banner
+			if len(banner.Format) < 1 && (banner.W == nil || *banner.W == 0 && banner.H == nil || *banner.H == 0) {
+				errs = append(errs, &errortypes.BadInput{
+					Message: "rubicon imps must have at least one imp.format element",
+				})
 				continue
 			}
-			bannerExt := rubiconBannerExt{RP: rubiconBannerExtRP{SizeID: primarySizeID, AltSizeIDs: altSizeIDs, MIME: "text/html"}}
+			bannerExt := rubiconBannerExt{RP: rubiconBannerExtRP{MIME: "text/html"}}
```

Comment:
With this new approach bannerExt can be extracted to constant and use it instead of Marshalling it every time.

Reply:
It's already in a separate variable and is being used only ones. What have I missed?

Reply:
This is an optimization suggestion.
you had:
```
bannerExt := rubiconBannerExt{RP: rubiconBannerExtRP{SizeID: primarySizeID, AltSizeIDs: altSizeIDs, MIME: "text/html"}}
```
where you set different variables to bannerExt

New change: 
```
bannerExt := rubiconBannerExt{RP: rubiconBannerExtRP{MIME: "text/html"}}
```
doesn't have any variables, it's always the same and has a hardcoded value. 

My idea was to prepare bannerExt for this case and set it to `bannerCopy.Ext`
Something like this:
```
bannerExt := []byte(`{"rp":{"mime":"text/html"}}`)  //this should be a constant outside of the function
bannerCopy.Ext = bannerExt
```